content/pt-pt/docs/home/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+approvers:
+- chenopis
```

Comment:
```suggestion
```
Only include reviewers and approvers if you are confident they speak (European) Portuguese